modules/ROOT/pages/vpn-about.adoc

```diff
@@ -0,0 +1,70 @@
+= Anypoint VPN
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint VPN feature enables you to create a secure connection to systems within your network that are hosted outside of Anypoint Platform. If required, you can create multiple VPNs to connect to different networks or different regions within a network.
```

Comment:
The "If required" construction leaves me confused--do I have to create multiple VPNs in order to connect to different networks or regions? Or is that optional? If it is optional, no need to mention it here. Maybe move this 2nd sentence to the note, so all that info is together in one place.